app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
```

Comment:
return dict not string

Reply:
`FAILED tests/test_main.py::test_print_animal_alive - TypeError: __repr__ returned non-string (type dict)`
**__repr__** should return a string only